test(btn): resolve .is-loading test flakiness

dancormier opened the pull request:
Currently, the test for `.s-btn.is-loading` fails in the GitHub Actions workflow but not locally (aka, _works on my machine_).

dancormier committed:
test(btn): renable .is-loading vis test

dancormier committed:
Rename s-btn test file

dancormier committed:
Merge branch 'develop' into dcormier/btn-loading-test-webkit

dancormier committed:
Merge branch 'develop' into dcormier/btn-loading-test-webkit

dancormier committed:
Uncomment s-btn vis regression test

dancormier committed:
Merge branch 'dcormier/btn-loading-test-webkit' of https://github.com/StackExchange/Stacks into dcormier/btn-loading-test-webkit

dancormier committed:
Try padded span wrapper

giamir commented:
@dancormier I had similar issues with the tooltip visual tests. The solution was to create a wrapper with a fixed height and width. If this happens often we could even extract the wrapper setup in a helper function:
https://github.com/StackExchange/Stacks/blob/develop/lib/test/s-tooltip.visual.test.ts#L12

dancormier commented:
@giamir ah, of course! I should've thought of that. Thanks!

dancormier committed:
Add fixed height/width wrapper

As suggested in https://github.com/StackExchange/Stacks/pull/1256#issuecomment-1415901446

dancormier committed:
Tweak dimensions

dancormier committed:
update baselines
